content/cpp/concepts/maps/terms/swap/swap.md

```diff
@@ -0,0 +1,127 @@
+---
+Title: '.swap()'
+Description: 'Exchanges the content of two maps.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Objects'
+  - 'OOP'
+  - 'Classes'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`.swap()`** function exchanges the content of two maps of the same type.
+
+## Syntax
+
+The `map1` is the map whose contents are exchanged with the contents of `map2`.
+
+```pseudo
+map1.swap(map2);
+```
+
```

Comment:
Please add the description for the parameters as well:
- `map1`: The first map whose contents will be swapped.
- `map2`: The second map to exchange contents with `map1`.